test/functionalTest/cases/2986_apiv2_notifications/apiv2_notification_reception.test

```diff
@@ -0,0 +1,304 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
General feedback regarding .test:

* In order to fully test the new parsing logic, I'd recommend a more systematic approach. I mean, to create an entity with an atribute of each posible type (text, number, bool, null, JSON vector, JSON object) and check that is correctly progressed to the second CB
* In would be great to include a 3-CBs federation scenario. Look for "federation" directories and/or filenames under cases/, I think there is some for 3-CBs federation sceanrio based in NGSIv1 that could be inspirational.

Reply:
> I mean, to create an entity with an atribute of each posible type

One of the attribute types could be geo:json, if you fell is a casa that deverses a special treatment beyond JSON array/object.

Reply:
Yeah, lots of tests missing for this PR, it's on my to-do list ...

Reply:
And about geo:json, I plan to include all types

Reply:
What do you mean "all types"? The ones I meandion + geo:json? Or also all the other geo:*? What about DateTime and ISO8601?

A bunch of it ;)

Reply:
Some addition in e56e601